teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization = 'no specialization yet', **args)
+    super(**args)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As the required for the project, the constructor method of the `Teacher` class to extend parent's constructor by adding `@classroom` and a parameter for it. Kindly ensure that to update it in order to match the expected requirements. It should look similar to the following instead:
```ruby
class Student < Person 
  def initialize(age, specialization, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @specialization = specialization
  end
  ...
```